CIP-DappConnector/CIP-dapp-connector.md

```diff
@@ -0,0 +1,250 @@
+---
+CIP: 30
+Title: Cardano dApp-Wallet Web Bridge
+Authors: rooooooooob
```

Comment:
Possibly, adding more authors here to include people who have actively contributed to the writing of this specification?

Reply:
What would we use for the threshold? Like @alessandrokonrad who has contributed significantly to discussion?